packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -163,6 +164,8 @@ export class PeoplePage {
 		await this.waitUntilLoaded();
 
 		await this.page.click( selectors.revokeInviteButton );
-		await this.page.waitForSelector( selectors.inviteRevokedMessage );
+
+		const noticeComponent = new NoticeComponent( this.page );
+		await noticeComponent.noticeShown( 'Invite deleted' );
```

Comment:
This change is not strictly related to the topic of this PR, but this is a good opportunity to update code that pre-date the new components.